packages/contracts/contracts/test/SecurityCouncil.t.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { SecurityCouncil_Initializer } from "./CommonTest.t.sol";
+
+contract SecurityCouncilTest is SecurityCouncil_Initializer {
+    /**
+     *  Events
+     */
+    event Confirmation(address indexed sender, uint256 indexed transactionId);
+    event Revocation(address indexed sender, uint256 indexed transactionId);
+    event Submission(uint256 indexed transactionId);
+    event Execution(uint256 indexed transactionId);
+    event ExecutionFailure(uint256 indexed transactionId);
+    event OwnerAddition(address indexed owner);
+    event OwnerRemoval(address indexed owner);
+    event RequirementChange(uint256 required);
+    event Validation(
+        uint256 indexed transactionId,
+        bytes32 outputRoot,
+        uint128 l2BlockNumber
+    );
+
+    struct Transaction {
+        address destination;
+        bool executed;
+        uint256 value;
+        bytes data;
+    }
+
+    struct OutputInfo {
+        bytes32 outputRoot;
+        uint128 l2BlockNumber;
+    }
+
+    function setUp() public virtual override {
+        super.setUp();
+    }
```

Comment:
You don't need these lines!

Reply:
That's right, this part is currently meaningless. Applied.